<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listItem</class>
 <widget class="QWidget" name="listItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_compName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string>航司名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_compPic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string>航司商标</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fliNo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
	color:rgb(0, 170, 255);
}
</string>
   </property>
   <property name="text">
    <string>航班号</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fliCate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
	color:rgb(0, 170, 255);
}
</string>
   </property>
   <property name="text">
    <string>航班型号</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_deparTime">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>出发时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_arrTime">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string>到达时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_price">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #0078d7;  /* 这是一种鲜亮的蓝色，可以根据需要调整颜色的具体数值 */
    font-size: 24px;  /* 字体大小，根据图片调整 */
    font-family: &quot;Segoe UI&quot;;  /* Windows 常用字体，可以换成其他的，如 'Arial' */
    font-weight: bold;  /* 加粗字体 */
	background-color:transparent;
}
</string>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_book">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid #FFA500;
    border-radius: 10px;
    background-color: #FFA500;
    color: white;
    padding: 5px 15px;
}

QPushButton:hover {
    background-color: #FF8C00;
    border-color: #E69500;
}

QPushButton:pressed {
    background-color: #CC8400;
    border-color: #B07300;
}</string>
   </property>
   <property name="text">
    <string>订票</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_dpAirPot">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>39</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string>出发机场</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_desAirPot">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>45</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string>到达机场</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>21</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">frame {
    background-color: transparent;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
